src/main/java/uk/gov/justice/probation/courtcaseservice/service/ImmutableCourtCaseService.java

```diff
@@ -142,7 +146,7 @@ void updateOtherProbationStatusForCrnByCaseId(String crn, String probationStatus
         if (crn != null) {
             final var courtCases = courtCaseRepository.findOtherCurrentCasesByCrnNotCaseId(crn, caseId)
                     .stream()
-                    .filter(courtCaseEntity -> !courtCaseEntity.getProbationStatus().equalsIgnoreCase(probationStatus))
+                    .filter(courtCaseEntity -> hasAnyDefendantsSameCrnDifferentProbationStatus(courtCaseEntity.getDefendants(), crn, probationStatus))
```

Comment:
Can't simply look in the court case entry for CRN. We need to look to see if any of the defendants have a CRN which matches but does NOT match the probation status.